bindings/jupyter-modules/lerna.json

```diff
@@ -0,0 +1,7 @@
+{
+  "lerna": "3.14.1",
+  "version": "0.5.2",
```

Comment:
Is it possible to share lerna.json to align the versioning between the component and its binding?